runtime/caml/intext.h

```diff
@@ -144,7 +143,7 @@ CAMLextern value caml_input_value_from_malloc(char * data, intnat ofs);
      to the beginning of the buffer, and [ofs] is the offset of the
      beginning of the externed data in this buffer.  The buffer is
      deallocated with [free] on return, or if an exception is raised. */
-CAMLextern value caml_input_value_from_block(const char * data, intnat len);
+CAMLextern value caml_input_value_from_block(char * data, intnat len);
```

Comment:
Why the removal of `const`?  `caml_input_value_from_block` is not supposed to change the bytes pointed to by the `data` argument.

Reply:
Fixed..